docs-web/configuration/security.md

```diff
@@ -0,0 +1,42 @@
+# Security
+
+We consider the security of the Ingress Controller paramount to the success of our users and use the following to 
+ensure the security of the Ingress Controller:
+* [Secure Development Life Cycle](https://www.microsoft.com/en-us/securityengineering/sdl/)
+* [FOSSA](https://fossa.com) scanning
+
+However, the Ingress Controller is deployed by a User in their environment, and as such, the User takes responsibility 
+for securing a *deployment* of the Ingress Controller. 
+We highly recommend every User to read and understand the following security concerns.
+
+## Kubernetes
+We recommend the Kubernetes [guide to securing a cluster](https://kubernetes.io/docs/tasks/administer-cluster/securing-a-cluster/).
+In addition, the following relating more specifically to Ingress Controller.
+
+### RBAC and Service Account
+The Ingress Controller is deployed within a Kubernetes environment, this environment must be secured. 
+Kubernetes uses [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) to control which types of users 
+can access which resources and what operations they can perform. 
+The Ingress Controller requires a service account which is configured using RBAC. 
+We strongly recommend using the [RBAC configuration](https://github.com/nginxinc/kubernetes-ingress/blob/master/deployments/rbac/rbac.yaml) provided in our standard deployment configuration. 
```

Comment:
I'm not sure about about the cases when a user would supply their own RBAC configuration which is not based on ours. I suggest rewriting it in way so that the user makes themselves aware of what resources the IC has access to:

We recommend inspecting the RBAC configuration (for [manifests installation](https://github.com/nginxinc/kubernetes-ingress/blob/master/deployments/rbac/rbac.yaml) or for [helm](https://github.com/nginxinc/kubernetes-ingress/blob/master/deployments/helm-chart/templates/rbac.yaml)) to understand what access the Ingress Controller service account has and to which resources. For example, by default the service account has access to all Secret resources in the cluster.

Reply:
updated.

Reply:
We advise the user to use the RBAC provided by us, as they can choose to override the config. It was a specific recommendation from the TMA meeting to make it clear that we're only responsible for the config we provide.